# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../ocfs2-tools/ocfs2-tools-1.6.4-alt-umode_t.patch
# Copyright (C) 2013 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

diff --git a/ocfs2-tools/include/ocfs2-kernel/ocfs2_fs.h b/ocfs2-tools/include/ocfs2-kernel/ocfs2_fs.h
index ffda634..031f227 100644
--- ocfs2-tools/include/ocfs2-kernel/ocfs2_fs.h
+++ ocfs2-tools/include/ocfs2-kernel/ocfs2_fs.h
@@ -25,6 +25,8 @@
 #ifndef _OCFS2_FS_H
 #define _OCFS2_FS_H
 
+typedef unsigned short umode_t;
+
 /* Version */
 #define OCFS2_MAJOR_REV_LEVEL		0
 #define OCFS2_MINOR_REV_LEVEL          	90
